tests/augmentation/test_augmentation.py

```diff
@@ -3881,6 +3847,22 @@ def test_module_sigma_tensor(self, device, dtype):
         img = torch.ones(1, 3, 5, 5, device=device, dtype=dtype)
         self.assert_close(op(img, *func_params), op_module(img))
 
+    @pytest.mark.slow
+    @pytest.mark.skipif(
+        torch_version_le(1, 13, 1),
+        reason="torch.compile is not available in previous versions",
+    )
+    def test_compile(self, device, dtype):
+        kernel_size = (3, 3)
+        sigma = (1.5, 2.1)
+        img = torch.rand(1, 3, 5, 5, device=device, dtype=dtype)
+        aug = RandomGaussianBlur(kernel_size, sigma, "replicate")
+        expected = aug(img)
+        aug = aug.compile(fullgraph=True)
+        actual = aug(img)
+        assert actual.shape == img.shape
+        self.assert_close(expected, actual)
```

Comment:
I wondered, shouldn't this fail due to the randomness of the augmentations?

Reply:
i'll check